data/remnant/remnant 1 introduction.txt

```diff
@@ -3741,4 +3742,73 @@ mission "Remnant: Expanded Horizons Storms 1"
+		has "remnant: blood test pure"
+		has "Remnant: Defense 1: done"
+		has "cargo space" >= 30
+	on offer
+		conversation
+			`You step out of the <ship> and look again at the strange sight of the <origin> spaceport. You still aren't used to a structure that looks as much like it grew from the landing pads as it was built next to them. The dock side entrance is busy; when your commlink vibrates, you have to find a place where the traffic eddies and you can step aside and learn that you need to check in with the information desk.`			
+			`	The woman working the desk seems startled to see you when you walk up. You can see the realization dawn on her as she scans your clothes. "Ah," she sings, "you are the ancestral sojourner among us."`
+			`	You are getting used to being gawked at, so you take it in good humor and respond by singing, "The Embers guided me to friends I didn't know, and your message invited me to meet a new one."`
+			`	Her hands make a short series of shakes and a moment later she remembers to laugh for you. The delayed response makes her laugh again, spontaneously this time, and you both join in. When she recovers herself, she sings, "Logistics have requested that we find someone who can carry 30 tons of cargo to <planet>. Is the <ship> available immediately?"`
+			choice
+				`	(Accept the job.)`
+					goto willdo
+				`	(Decline the job.)`
+					goto notinterested
+			label willdo
+			`	"Certainly," you respond. The job board has been stubbornly empty when you've checked, so a rush job appearing intrigues you.`
+			`	"Excellent," she responds in a full alto, "I will let the project manager know the cargo will arrive promptly." She signs what you think is a farewell and turns to address someone else who has stepped up.`
+			`	You manage to chant and point your way through the cafeteria line, and you are aware of the flurry of signs around you. There doesn't seem to be a way to pay, so you find a table and sit alone until your commlink chimes to let you know that the <ship> has been refueled and the mystery cargo loaded.`
+				accept
+			label notinterested
+			`	"That is unfortunate," she sings as she makes a notation on her terminal. "But there will be others who are willing to help."`
+				decline
+	on complete
+		payment
+		conversation
+			`You step off the <ship> in <planet> to see a man wandering around your ship taking video with a datapad and softly dictating some running commentary. He dodges around the dock workers unloading and refueling, seemingly intent on recording every aspect of your ship.`
+			`	"Cousin!" he sings with excitement, when you walk up to him. "It is good to meet you and to see your wonderful ship. I've only seen such designs in our historical entertainments." He holds up the datapad facing you and says, I am making a video for my companion. He will not believe me unless I have proof, and maybe not even then!" he laughs. You wave awkwardly at the camera and are relieved when he stops recording.`  
```

Comment:
> One problem here: There's not guarantee that the player is flying a ship from ancestral human space. If it triggers soon after the player meets these conditions then they likely are, but they _might_ not get this mission until after they have earned their license and are flying something like a gull or starling.

I can't find a way to determine what kind of ship the player is flying, so I can't be too granular in approaching this. I've adjusted the conversation to be broader than ancestral (e.g. Hai ships), but it will still read oddly if they have a Remnant ship.

Reply:
That's probably the best we can do. There isn't, as far as I can tell, a way to pinpoint specific ships (or families of ships), although I really wish there was.